PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,51 @@
+<!-- Thank you for submitting this PR! :) -->
+<!-- Provide a general summary of your changes in the Title above ^, end with (close #<issue-no>) or (fix #<issue-no>) -->
+
+### Description
+<!-- The title might not be enough to convey how this change affects the user. -->
+<!-- Describe the changes from a user's perspective -->
+
+### Changelog
```

Comment:
In the learn course template, we don't have versions or changelog. Can we remove this section?